src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1102,6 +1150,12 @@ protected static ImmutableArray<string> CheckTupleElementNames(int cardinality,
                                                     nameof(memberIsReadOnly), nameof(memberNames)));
             }
 
+            if (!memberNullableAnnotations.IsDefault && memberNullableAnnotations.Length != memberTypes.Length)
+            {
+                throw new ArgumentException(string.Format(CodeAnalysisResources.AnonymousTypeArgumentCountMismatch2,
+                                                    nameof(memberNullableAnnotations), nameof(memberNames)));
```

Comment:
Nit: indent to be even with the other argument in the method.

Reply:
This matches the other examples above.

---
In reply to: [302305173](https://github.com/dotnet/roslyn/pull/37048#discussion_r302305173) [](ancestors = 302305173)

Reply:
Ow, but ok

---
In reply to: [302364090](https://github.com/dotnet/roslyn/pull/37048#discussion_r302364090) [](ancestors = 302364090,302305173)